tuftool/Cargo.toml

```diff
@@ -35,7 +35,7 @@ simplelog = "0.11"
 snafu = { version = "0.7", features = ["backtraces-impl-backtrace-crate"] }
 structopt = "0.3"
 tempfile = "3.3.0"
-tokio = "~1.8"  # LTS
+tokio = "1.8"
```

Comment:
Since it's a binary target, it seems like we should leave `tuftool` pinned to ensure that it gets the dependencies we expect.

Reply:
We could. I'll make the change.